tests/common/blocks.rs

```diff
@@ -1,114 +0,0 @@
-
-    println!("contract addresses:");
-    println!("\terc20(eth): {}", to_felt252(block_context.fee_token_address(&FeeType::Eth).0.key()));
-    println!("\terc20(strk): {}", to_felt252(block_context.fee_token_address(&FeeType::Strk).0.key()));
-    println!("\taccount: {}", to_felt252(account_address.0.key()));
-    println!("\tcontract: {}", to_felt252(contract_address.0.key()));
-
-    let account_tx = test_utils::account_invoke_tx(invoke_tx_args! {
-        max_fee,
-        sender_address: account_address,
-        calldata: create_calldata(
-            contract_address,
-            "return_result",
-            &[stark_felt!(2_u8)],
-        ),
-        version: tx_version,
-        nonce: nonce_manager.next(account_address),
-        only_query,
-    });
-
-    let initial_state = copy_state(&state);
-
-    let account_tx_internal = to_internal_tx(&account_tx);
-
-    let tx_execution_info = account_tx.execute(&mut state, &block_context, true, true).unwrap();
-
-    os_hints(&block_context, initial_state, vec![account_tx_internal], vec![tx_execution_info])
-}
-
-#[fixture]
-pub fn simple_block_cairo1(
-    block_context: BlockContext,
-    max_fee: Fee,
-    #[default(CairoVersion::Cairo0)] cairo_version: CairoVersion,
-    #[default(TransactionVersion::ZERO)] tx_version: TransactionVersion,
-    #[default(false)] only_query: bool,
-) -> (StarknetOsInput, ExecutionHelperWrapper) {
-    let account = FeatureContract::AccountWithoutValidations(CairoVersion::Cairo1);
-    let test_contract = FeatureContract::TestContract(cairo_version);
-    let erc20 = FeatureContract::ERC20;
-
-    let mut state = test_state(&block_context, BALANCE, &[(account, 1), (erc20, 1), (test_contract, 1)]);
-
-    let account_address = account.get_instance_address(0);
-    let contract_address = test_contract.get_instance_address(0);
-    let mut nonce_manager = NonceManager::default();
-
-    println!("contract addresses:");
-    println!("\terc20(eth): {}", to_felt252(block_context.fee_token_address(&FeeType::Eth).0.key()));
-    println!("\terc20(strk): {}", to_felt252(block_context.fee_token_address(&FeeType::Strk).0.key()));
-    println!("\taccount: {}", to_felt252(account_address.0.key()));
-    println!("\tcontract: {}", to_felt252(contract_address.0.key()));
-
-    let account_tx = test_utils::account_invoke_tx(invoke_tx_args! {
-        max_fee,
-        sender_address: account_address,
-        calldata: create_calldata(
-            contract_address,
-            "return_result",
-            &[stark_felt!(2_u8)],
-        ),
-        version: tx_version,
-        nonce: nonce_manager.next(account_address),
-        only_query,
-    });
-
-    let initial_state = copy_state(&state);
-
-    let account_tx_intenal = to_internal_tx(&account_tx);
-
-    let tx_execution_info = account_tx.execute(&mut state, &block_context, true, true).unwrap();
-
-    os_hints(&block_context, initial_state, vec![account_tx_intenal], vec![tx_execution_info])
-}
```

Comment:
I don't understand the point of not using fixtures.

Reply:
There is a fixture logic which prepares some state for testing and a test logic which depends of what one wants to check in this particular test. Imho, block content (set of transactions) is specific to the particular test.

Reply:
Okay, fine by me then.